scripts/exgdas_enkf_select_obs.sh

```diff
@@ -72,7 +72,7 @@ cd $DATA || exit 8
 ################################################################################
 # ObsInput file from ensemble mean
 rm -f obs_input*
```

Comment:
@RussTreadon-NOAA 
Should there also be a "rm -f obsinput.tar" here? The exglobal_atmos_analysis.sh script has both "rm obs_input.*" and "rm obsinput.tar".

Reply:
`rm -f obsinput.tar` has been added.  Good suggestion.